### PR TITLE
Gib Interface

### PR DESCRIPTION
**Pull request by ThatDan123**
-Adds a gib interface

-Blobs and Ai can now be smited

-Adds drop all function for dynamic item storage

### CODE REVIEW

File: UnityProject/Assets/Scripts/Managers/CargoManager.cs [hunk 1]
```diff
@@ -262,11 +263,11 @@ public void OnShuttleArrival()
 
 		public void ProcessCargo(GameObject obj, HashSet<GameObject> alreadySold)
 		{
-			if (obj.TryGetComponent<PlayerScript>(out var playerScript))
+			if (obj.TryGetComponent<IGib>(out var iGib))
 			{
 				// No one must survive to tell the secrets of Central Command's cargo handling techniques.
 				Chat.AddExamineMsg(obj, "<color=red> You feel a strong force of energy run through your body before everything goes to black in the blink of the eye. </color>");
-				playerScript.playerHealth.Gib();
```
Comment (NoooneyDude) [inline UnityProject/Assets/Scripts/Managers/CargoManager.cs:269]: Now I'm curious - I don't suppose the cargo shuttle tries to sell the AI eye?

Comment (ThatDan123) [reply UnityProject/Assets/Scripts/Managers/CargoManager.cs:269]: huh, thats a good point

**Review by NoooneyDude — DISMISSED** [review]

**Review by NoooneyDude — DISMISSED** [review]

**Comment by MaxIsJoe** [comment]
Beat me to it Dan, thanks 👍 

also the PR number, Nice.

File: UnityProject/Assets/Scripts/HealthV2/IGib.cs [hunk 2]
```diff
@@ -0,0 +1,7 @@
+﻿namespace HealthV2
+{
+	public interface IGib
```
Comment (corp-0) [inline UnityProject/Assets/Scripts/HealthV2/IGib.cs:3]: is ``IGibable`` too stupid of a name?

Comment (AtnerNT) [reply UnityProject/Assets/Scripts/HealthV2/IGib.cs:3]: ICanBeGibbed?

**Review by corp-0 — APPROVED** [review]

**Pull request merged**